fastlane/Fastfile

```diff
@@ -103,10 +101,28 @@ platform :ios do
         file_path = "../ios/#{language}.lproj/InfoPlist.strings"
         lines = File.readlines(file_path)
 
-        updated_lines = lines.reject { |line| keys.any? { |key| line.start_with?("\"#{key}\"") } }
+        lines_hash = {}
+        lines.each do |line|
+          match = line.match(/^"(.*?)"\s*=\s*"(.*?)";$/)
+          if match
+            key = match[1]
+            value = match[2]
+            lines_hash[key] = value
+          end
+        end
 
-        File.open(file_path, 'w') do |file|
-          file.puts(updated_lines)
+        # Apple requires that the key NSLocationAlwaysAndWhenInUseUsageDescription is present even if the app only uses when in use
+        # because some of the libraries we use require always permission even if we don't use it.
```

Comment:
> some of the libraries we use require always permission

Not actually connected to your changes, but this made me check the App Store to see what permissions is displayed there, and sure enough, they do say that we could use location in the background for 1.45.

![Screenshot 2024-01-30 at 14 00 00](https://github.com/AtB-AS/mittatb-app/assets/1774972/875fa4ec-638f-4420-bb0b-fcb3f08ff9bf)

Not sure how that works, but is it something we can change about the related libraries?

Reply:
Same applies to FRAM / NFK

Reply:
In this PR the background location was removed, I think it might change that message as well, I hope.